lib/os_allocation.c

```diff
@@ -1615,7 +1615,7 @@ os_mba_set(const unsigned socket,
 }
 
 int
-os_mba_get(const unsigned socket,
+os_mba_get(const unsigned mba_id,
```

Comment:
Please update header file

Reply:
yes. again.. These are taken care in further patches. I will have to add further patches if need to fix it.

Reply:
This is very small change in os_allocation.h file header that can be included in this pull request

Reply:
ok.  Took care of it.